src/forcemove.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+#include "move.h"
+#include "energy.h"
+#include "units.h"
+
+namespace Faunus {
+
+namespace Move {
+
+/**
+ * @brief Base class for dynamics integrators
+ *
```

Comment:
Add a little bit more info about the responsibilities

Reply:
Done in a short introductory docblock.